lib/calendar.js

```diff
@@ -0,0 +1,114 @@
+'use strict';
+
+class CalendarEvent {
+  static toICS(json) {
+    const data = JSON.parse(json);
+    const handleBlock = (block, result = []) => {
+      for (const [key, value] of Object.entries(block)) {
+        if (typeof value === 'string') {
+          const LINE_LENGTH = 80;
+          if (value.length >= LINE_LENGTH) {
+            result.push(`${key}:${value.substring(0, LINE_LENGTH).trim()}`);
+            for (let i = LINE_LENGTH; i <= value.length; i += LINE_LENGTH - 1) {
+              const nextLine = value.substring(i, i + LINE_LENGTH - 1).trim();
+              if (!nextLine) continue;
+              result.push(' ' + nextLine);
+            }
+            continue;
+          }
+          result.push(`${key}:${value}`);
+          continue;
+        }
+
+        if (Array.isArray(value)) {
+          for (const item of value) {
+            result.push(`BEGIN:${key}`);
+            const content = handleBlock(item);
+            result.push(...content);
+            result.push(`END:${key}`);
+          }
+          continue;
+        }
+
+        if (typeof value === 'object') {
+          if (value.parameters && value.value) {
+            const params = Object.entries(value.parameters)
+              .map(([paramKey, paramValue]) => `${paramKey}=${paramValue}`)
+              .join(';');
+            result.push(`${key};${params}:${value.value}`);
+            continue;
+          }
+          let line = key + ':';
+          for (const [paramKey, paramValue] of Object.entries(value)) {
+            line += `${paramKey}=${paramValue};`;
+          }
+          line = line.slice(0, -1);
+          result.push(line);
+          continue;
+        }
+      }
+      return result;
+    };
+
+    const ics = handleBlock(data);
+    return ics.join('\n');
+  }
+  static toJSON(ics) {
```

Comment:
We have class with 2 static methods, it would be better to have 2 functions, we do not need class at all